src/providers/theme/Provider.tsx

```diff
@@ -0,0 +1,15 @@
+import { useState } from "react";
+import React from "react";
+import { ThemeContext } from "./Context";
+
+type ProviderProps = {
+  children: React.ReactNode;
+};
+
+function ThemeProvider({ children }: ProviderProps): JSX.Element {
+  const [lottery, setLottery] = useState("mega-sena");
```

Comment:
We shouldn't tie themes with lotteries here, you may want to rename it to `theme, setTheme`, also use "default" for the default value.